src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs

```diff
@@ -58,9 +58,22 @@ private bool ShouldShowLightDismissOverlay
 
 		public Popup()
 		{
+			this.RegisterDefaultValueProvider(GetDefaultValue);
 			Initialize();
 		}
 
+		private bool GetDefaultValue(DependencyProperty dependencyProperty, out object value)
+		{
+			value = null;			
+
+			if (dependencyProperty == IsLightDismissEnabledProperty)
+			{
+				value = FeatureConfiguration.Popup.EnableLightDismissByDefault ? true : false;
```

Comment:
Would this work the same?

```suggestion
				value = FeatureConfiguration.Popup.EnableLightDismissByDefault;
```

Reply:
Lol, of course, I'm stupid 😂😂😂 !

Reply:
@MartinZikmund FYI there are cases where the `? true : false` is desired due to a runtime bug (performance issue) (https://github.com/dotnet/runtime/issues/37904) but that seems to happen only when generics is involved.